tests/Integration/TestCase.php

```diff
@@ -37,17 +37,18 @@ abstract class TestCase extends WPIntegrationTestCase {
 	 *
 	 * @param array $custom_options Associative array of option keys and values that should be saved.
 	 */
-	public static function set_options( $custom_options = array() ): void {
+	public static function set_options( array $custom_options = array() ): void {
 		update_option( \Parsely::OPTIONS_KEY, array_merge( self::DEFAULT_OPTIONS, $custom_options ) );
 	}
 
 	/**
 	 * Create a test post.
 	 *
 	 * @param string $post_type Optional. Post type. Default is 'post'.
+	 *
 	 * @return array An array of WP_Post fields.
 	 */
-	public function create_test_post_array( $post_type = 'post' ): array {
+	public function create_test_post_array( string $post_type = 'post' ): array {
```

Comment:
What benefit does the method serve? Only one value is overrideable (and a post title saying "Post" wouldn't make sense when the post type is not `post`), so it gives less flexibility than using the [default test post values](https://core.trac.wordpress.org/browser/trunk/tests/phpunit/includes/factory/class-wp-unittest-factory-for-post.php#L17) with the factory `create()` and similar functions.

We may want to think about scrapping it.